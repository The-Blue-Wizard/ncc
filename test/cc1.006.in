/* struct init: brace elision */

#include "support.h"

struct x
{
    int a, b;
} x[] = {
    { 1, 2 },
    3, 4, 
    { 3 },
    5
};

#define NR_X (sizeof(x)/sizeof(*x))

int
main(void)
{
    int i;

    for (i = 0; i < NR_X; ++i)
        putobj(&x[i], sizeof(struct x));

    return 0;
}

