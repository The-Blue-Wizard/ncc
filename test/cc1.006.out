0100000002000000
0300000004000000
0300000000000000
0500000000000000
